pan/types.pan

```diff
@@ -48,12 +48,13 @@ Ex: 20040825120123Z
 }
 function is_asndate = {
     # Check cardinality and type of argument.
-    if (ARGC != 1 || !is_string(ARGV[0]))
-        error("usage: is_asndate(string)");
+    if (ARGC != 1 || !is_string(ARGV[0])) error("usage: is_asndate(string)");
```

Comment:
Really not a fan of this style. 

```
if (...) 
    # action;
```

is much more readable most of the time IMHO, especially when the $condition is contains multiple operators. As a code reader I want to be able to focus on the logic and be able to visually filter out the resulting action.

Reply:
Would the following be okay?
```pan
if (...) {
    # action;
};
```

Reply:
Even better.